mack/__init__.py

```diff
@@ -21,21 +21,21 @@ def type_2_scd_upsert(path, updates_df, primary_key, attr_col_names):
 
 
 def type_2_scd_generic_upsert(
-        path,
-        updates_df,
-        primary_key,
-        attr_col_names,
-        is_current_col_name,
-        effective_time_col_name,
-        end_time_col_name,
+    path,
```

Comment:
@souvik-databricks @robertkossendey - your Black formatting settings are fighting with each other.  How are you both running black?  `poetry run black .`?

Reply:
I use the pre-commit hooks. It will run automatically after executing `pre-commit install` once.